pkgs/os-specific/darwin/apple-source-releases/default.nix

```diff
@@ -220,17 +220,22 @@ let
   macosPackages_11_0_1 = import ./macos-11.0.1.nix { inherit applePackage'; };
   developerToolsPackages_11_3_1 = import ./developer-tools-11.3.1.nix { inherit applePackage'; };
 
-  applePackage' = namePath: version: sdkName: sha256:
+  applePackage'_ = stdenv: namePath: version: sdkName: sha256:
```

Comment:
Probably don't need to introduce a function that we only call once

Reply:
True.